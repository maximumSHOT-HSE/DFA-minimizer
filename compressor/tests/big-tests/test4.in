14 70 5
1 9 0
1 13 1
1 5 2
1 6 3
1 3 4
2 7 0
2 3 1
2 1 2
2 7 3
2 3 4
3 8 0
3 12 1
3 7 2
3 10 3
3 9 4
4 4 0
4 14 1
4 10 2
4 6 3
4 9 4
5 11 0
5 10 1
5 1 2
5 9 3
5 13 4
6 1 0
6 12 1
6 11 2
6 9 3
6 11 4
7 6 0
7 5 1
7 14 2
7 7 3
7 11 4
8 9 0
8 3 1
8 2 2
8 9 3
8 10 4
9 14 0
9 8 1
9 14 2
9 13 3
9 11 4
10 12 0
10 14 1
10 2 2
10 9 3
10 8 4
11 9 0
11 13 1
11 7 2
11 1 3
11 4 4
12 4 0
12 7 1
12 14 2
12 5 3
12 7 4
13 9 0
13 3 1
13 5 2
13 7 3
13 6 4
14 7 0
14 1 1
14 12 2
14 5 3
14 5 4
10
6
2 14 1 9 12 3 
