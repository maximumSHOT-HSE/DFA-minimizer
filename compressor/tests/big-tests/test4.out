14 70 5
6 1 0
14 1 1
2 1 2
5 1 2
11 1 3
8 2 2
10 2 2
2 3 1
8 3 1
13 3 1
1 3 4
2 3 4
4 4 0
12 4 0
11 4 4
7 5 1
1 5 2
13 5 2
12 5 3
14 5 3
14 5 4
7 6 0
1 6 3
4 6 3
13 6 4
2 7 0
14 7 0
12 7 1
3 7 2
11 7 2
2 7 3
7 7 3
13 7 3
12 7 4
3 8 0
9 8 1
10 8 4
1 9 0
8 9 0
11 9 0
13 9 0
5 9 3
6 9 3
8 9 3
10 9 3
3 9 4
4 9 4
5 10 1
4 10 2
3 10 3
8 10 4
5 11 0
6 11 2
6 11 4
7 11 4
9 11 4
10 12 0
3 12 1
6 12 1
14 12 2
1 13 1
11 13 1
9 13 3
5 13 4
9 14 0
4 14 1
10 14 1
7 14 2
9 14 2
12 14 2
10
6
1 2 3 9 12 14 
